RadixWallet/Features/HomeFeature/Coordinator/Home+View.swift

```diff
@@ -88,6 +80,56 @@ extension Home {
 	}
 }
 
+@MainActor
+private extension View {
+	func destinations(_ destinationStore: PresentationStoreOf<Home.Destination>) -> some SwiftUI.View {
+		accountDetails(destinationStore)
+			.createAccount(destinationStore)
+			.exportMnemonic(destinationStore)
+			.importMnemonics(destinationStore)
+	}
+
+	func accountDetails(_ destinationStore: PresentationStoreOf<Home.Destination>) -> some SwiftUI.View {
+		navigationDestination(
+			store: destinationStore,
+			state: /Home.Destination.State.accountDetails,
+			action: Home.Destination.Action.accountDetails,
+			destination: { AccountDetails.View(store: $0) }
+		)
+	}
+
+	func createAccount(_ destinationStore: PresentationStoreOf<Home.Destination>) -> some SwiftUI.View {
+		sheet(
+			store: destinationStore,
+			state: /Home.Destination.State.createAccount,
+			action: Home.Destination.Action.createAccount,
+			content: { CreateAccountCoordinator.View(store: $0) }
+		)
+	}
+
+	func exportMnemonic(_ destinationStore: PresentationStoreOf<Home.Destination>) -> some SwiftUI.View {
+		fullScreenCover( /* Full Screen cover to not be able to use iOS dismiss gestures */
```

Comment:
`/* Full Screen cover to not be able to use iOS dismiss gestures */` - No need to have full screen cover for that, we can set `.interactiveDismissDisabled(true)` on the sheet to disable dismiss gesture.

Reply:
True, but we use fullScreenCover in import mnemonics flow, where it fixes a bug, and one might like consistency. Will take a look